privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -329,51 +423,55 @@ def prediction_on_specific_property(
 
         return prediction
 
-    def attack(self):
+    def attack(self) -> Tuple[str, Dict[str, float]]:
         """
         Perform Property Inference attack.
```

Comment:
This is nice - starts with a verb. Usually you would say "perform**s**" since it is "it" = the method